algorithms/kernel/cosdistance/cosdistance_lp_impl.i

```diff
@@ -69,7 +69,7 @@ services::Status cosDistanceLowerPacked(const NumericTable * xTable, NumericTabl
 
         /* compute inverse of sqrt of gemm result and save for use in computation off-diagonal blocks */
         PRAGMA_VECTOR_ALWAYS
-        for (int i = 0; i < blockSize1; i++)
+        for (DAAL_INT i = 0; i < blockSize1; i++)
```

Comment:
seems 'size_t' would be more preferable here.

Reply:
Yep, but intention of this patch is to make it consistent. Changing it could lead to huge code refactoring and I not sure that it should be implemented in such context.